src/ui/popup/App.tsx

```diff
@@ -128,7 +128,11 @@ class App extends React.Component<PopupAppComponentProps, InitialState> {
   ): Partial<HTMLButtonElement> {
     return (
       <button
```

Comment:
Perhaps it's best to move this group out to a separate file to keep the main App.tsx file cleaner.

Reply:
We would need to then make some of the functions inside the App importable then as the click function and animate are also used outside of this HTML function